Time/rescuetime-activities.3m.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env PYTHONIOENCODING=UTF-8 python3
```

Comment:
This change allows python to be picked up from a variety of places -- macOS install, Homebrew on Intel silicon install, Homebrew on M1 silicon install, etc.

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" PYTHONIOENCODING=UTF-8 python3
```

Reply:
Changed this line slightly (to prioritize custom Python installations).